graph/graph.go

```diff
@@ -174,6 +179,20 @@ func (graph *Graph) restore() error {
 	for _, v := range dir {
 		id := v.Name()
 		if graph.driver.Exists(id) {
+			pth := filepath.Join(graph.root, id, "json")
```

Comment:
Why not `graph.loadImage(id)`? After #16953 the parent reference may not be in the raw json file. "json" is a constant `jsonFileName`.